javagen/src/main/java/com/azure/autorest/util/ClientModelUtil.java

```diff
@@ -714,4 +714,83 @@ public static Set<String> getExternalPackageNamesUsedInClient(List<ClientModel>
 
         return externalPackageNames;
     }
+
+    /**
+     * Gets or creates a new FileDetails model for a multipart/form-data request
+     *
+     * @param compositeType the object schema of the multipart/form-data request model.
+     * @param filePropertyName the property name of the file in the multipart/form-data request model.
+     * @return the ##FileDetails model
+     */
+    public static IType getMultipartFileDetailsModel(
+            ObjectSchema compositeType,
+            String filePropertyName) {
+        // TODO (weidxu): this ##FileDetails model may get renamed and moved to azure-core
```

Comment:
Wouldn't it cause breaking change if moved to azure-core?

Reply:
Yes, it would.

The issue is that DocumentTranslation need it this week. And one won't have azure-core thing designed and merged that soon.

So, at present, a model in SDK. The decision about azure-core would be later, but should be before any of them get GA.

Reply:
I see.